src/components/Header/Header.scss

```diff
@@ -1,7 +1,192 @@
+$header-height-desktop: 64px;
+$header-height-tablet: 48px;
+
+.header {
+  display: flex;
+  box-shadow: 0px 1px 0px 0px #e2e6e9;
+  @include on-tablet {
+    height: $header-height-tablet;
+  }
+
+  @include on-desktop {
+    height: $header-height-desktop;
+  }
+
+  .container {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    box-sizing: border-box;
+
+    @include on-tablet {
+      height: $header-height-tablet;
+    }
+
+    @include on-desktop {
+      height: $header-height-desktop;
+    }
+  }
+
+  &__logo {
+    display: flex;
+    box-sizing: border-box;
+
+    @include on-phone {
+      padding: 13px 16px;
+    }
+
+    @include on-tablet {
+      padding: 13px 16px;
+    }
+
+    @include on-desktop {
+      padding: 18px 24px;
+    }
+  }
+
+  &__left {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    height: 100%;
+    box-sizing: border-box;
+  }
+
+  @include on-tablet {
+    gap: 16px;
+  }
+
+  @include on-desktop {
+    gap: 24px;
+  }
+}
+
 .nav {
-  position: sticky;
-  top: 0;
-  text-align: center;
-  flex-grow: 0;
-  flex-shrink: 0;
+  display: flex;
+  justify-content: flex-start;
+  height: 100%;
+  box-sizing: border-box;
+
+  &__list {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+
+    @include on-tablet {
+      gap: 32px;
+    }
+
+    @include on-desktop {
+      gap: 64px;
+    }
+
+    @media (max-width: 639px) {
+      display: none;
+    }
+  }
+
+  &__link {
+    display: block;
+
+    font-size: 12px;
+    line-height: 14px;
+    font-weight: 800;
+
+    text-transform: uppercase;
+    color: #89939a;
+
+    height: 100%;
+
+    @include on-tablet {
+      padding: 17px 0;
+    }
+
+    @include on-desktop {
+      padding: 25px 0;
+    }
+
+    transition: color 0.3s;
+
+    &:hover {
+      color: #313237;
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+
+  color: #313237;
+
+  &::after {
+    content: '';
+    position: absolute;
+    width: 100%;
+    height: 4px;
+    background-color: #313237;
+    border-radius: 8px;
+    bottom: 0;
+    left: 0;
+    right: 0;
+  }
+}
+
+.icons {
+  display: flex;
+  box-shadow: -1px 0 0 0 #e2e6e9;
+
+  @include on-tablet {
+    padding: 16px;
+    gap: 32px;
+  }
+
+  @include on-desktop {
+    gap: 48px;
+    padding: 24px;
+  }
+
+  @media (max-width: 639px) {
+    display: none;
+  }
+}
+
+.icon {
+  display: flex;
+  align-items: center;
+  width: 16px;
+  height: 16px;
+
+  &--heart {
+    background-image: url(../Header/heartLike.svg);
+  }
+
+  &--cart {
+    background-image: url(../Header/cart.svg);
+  }
+}
+
+.burger-menu {
+  display: none;
+  @include on-phone {
+    display: flex;
+    box-shadow: -1px 0 0 0 #e2e6e9;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 24px;
+    height: 24px;
+    padding: 16px;
+    cursor: pointer;
+
+    span {
```

Comment:
What span is this? Why it doesn't have it own css class with name, that refers to its functionality?